### DISCUSSION TITLE
Get Api For DiagramGPT

### DISCUSSION THREAD

**Discussion by UsamaYaseen-DataScience** [deepgram/community#763] [General help]
I want to integrate the DIAGRAMGPT with my colab notebook where i user gave input and they display the diagram how we do this anyone ?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by UsamaYaseen-DataScience** [reply]
i try to generate key for DiagramGPT but modle not find in key generation place ?

**Comment by SandraRodgers** [comment]
Do you have a question related to Deepgram?
